### ISSUE TITLE
Allow for $ characters in password / extra field of Airflow connections

### ISSUE THREAD

**Issue by MarvinSchenkel** [airflow-helm/charts#14]
**What is the bug?**
Whenever I try adding an Airflow connection that has a $-character in either the `password` or `extra` field, the value gets chopped off. I suspect this is because ultimately the value is passed to a bash command (`airflow connections --add`) that interprets $ as a bash-variable, which is obviously empty. For example, if I submit an airflow connection with password / extra 'abcd$ef', the password / extra stored in Airflow only contains 'abcd', since the variable $ef is not defined. If I manually escape the $-character with a \ in the bash-command, the value is properly added.

**What are your Helm values?**
```yaml
scheduler:
  connections:
    - id: 'test'
      type: http
      login: 'test'
      password: 'A^Ci$c#h"GhjCofpn\R$'
      extra: 'A^Ci$c#h"GhjCofpn\R$'
```

**What is your Kubernetes Version?**:
1.17.13

**What is your Helm version?**:
3.4.1

**Comment by thesuperzapper** [comment]
@MarvinSchenkel this is resolved in versions after `7.14.3`

**Comment by MarvinSchenkel** [comment]
Thanks a bunch, @thesuperzapper !
